src/test/java/org/broadinstitute/consent/http/db/DataAccessRequestDAOTest.java

```diff
@@ -826,34 +814,203 @@ void createProgressReport() {
   }
 
   @Test
-  void testFindDARsByDateRange() throws InterruptedException {
-    DarCollection darCollection = createDarCollection();
-    darCollection.getDars().keySet().forEach((referenceId) ->
-      dataAccessRequestDAO.updateDraftToSubmittedForCollection(darCollection.getDarCollectionId(),
-          referenceId));
-    Timestamp oneYearAgo = Timestamp.from(Instant.now().minus(365, ChronoUnit.DAYS));
-    Timestamp oneMinuteInTheFuture = Timestamp.from(Instant.now().plus(1, ChronoUnit.MINUTES));
-    List<DataAccessRequest> dars = dataAccessRequestDAO.findSubmittedDarsByTimeRange(oneYearAgo,
-        oneMinuteInTheFuture);
+  void testFindAgedDARsByEmailTypeOlderThanInterval() throws InterruptedException {
+    User userOne = createUserWithInstitution();
+    Integer userOneId = userOne.getUserId();
+    Integer collectionOneId = createDarCollection(userOneId);
+    Integer collectionTwoId = createDarCollection(userOneId);
+    createDataAccessRequest(collectionOneId, userOneId,
+        new Date(Instant.now().minus(360,
+            ChronoUnit.DAYS).toEpochMilli()));
+    createDataAccessRequest(collectionTwoId, userOneId,
+        new Date(Instant.now().minus(15,
+            ChronoUnit.DAYS).toEpochMilli()));
+
+    List<DataAccessRequest> dars = dataAccessRequestDAO.findAgedDARsByEmailTypeOlderThanInterval(
+        EmailType.DAR_EXPIRATION_REMINDER.getTypeInt(),
+        "11 months", Timestamp.from(Instant.now().minus(365, ChronoUnit.DAYS)));
```

Comment:
I wonder if there's a better way to codify the interval string. Maybe something like a custom Interval object ... `new DuosInterval(11, ChronoUnit.MONTHS.name).toString()`?

Reply:
I'm making a static variable for each string for now.  Let me know if that satisfies this for now.